popdat/src/lib.rs

```diff
@@ -119,13 +119,11 @@ pub fn generate_scenario(
 
     let mut scenario = Scenario::empty(map, scenario_name);
     timer.start("building people");
-    for person in people {
-        // TODO If we need to parallelize because make_person is slow, the sim crate has a fork_rng
-        // method that could be useful
-        scenario
-            .people
-            .push(make_person::make_person(person, map, rng, &config));
-    }
+
+    scenario.people.append(&mut make_person::make_people(
```

Comment:
The API of [append](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.append) is funny, I would expect it to consume...

Reply:
This is mostly just my lack of fluency in rust.

I *think* `append` is intended for things where it makes sense to clear out but re-use the buffer like:

```
let buffer = vec![];
while read_into_buffer(&mut buffer) {
    results.append(buffer);
}
```

I've switched to `extend` which has better semantics for this type of call.